components/match2/wikis/dota2/matchgroup_input_custom.lua

```diff
@@ -47,7 +46,7 @@ local _NOT_PLAYED_SCORE = -1
 local _NO_WINNER = -1
 local _SECONDS_UNTIL_FINISHED_EXACT = 30800
 local _SECONDS_UNTIL_FINISHED_NOT_EXACT = 86400
-local _MIN_EARNINGS_FOR_FEATURED = 100000
+local _MIN_EARNINGS_FOR_FEATURED = 50000
```

Comment:
this change was discussed with the contributors?

Reply:
No, i changed it for testing. Just forgot.